services/shhext/api.go

```diff
@@ -308,6 +309,7 @@ func (api *PublicAPI) SendDirectMessage(ctx context.Context, msg chat.SendDirect
 		response = append(response, hash)
 
 	}
+	api.log.Info("response", "response", response)
```

Comment:
Was this checked in by mistake? If it's just a temporary helper, we can add a TODO to remember to remove it at the end.